src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.springboot.datajpa.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
```

Comment:
Lombok's `@Data` is very dangerous with entity classes
see here why
https://habr.com/ru/company/haulmont/blog/564682/